tests/libYARP_math/MathTest.cpp

```diff
@@ -539,7 +539,160 @@ class MathTest : public UnitTest {
         divisionOperator();
         crossProduct();
         quaternionConversion();
+        eulerTests();
+        signTest();
+        eigenTest();
+        elementTest();
+        catAndPileTest();
     }
+
+    //tests written by Giulia V
```

Comment:
Sorry,
You're perfectly right both about the comment and the eigenValues
computation. The result that I used for the comparison shows the eigen
values of [0 2; -2 -4], instead of [-1 2; -2 -4]
I'll correct my code asap :)
Il 27/Feb/2016 14:59, "Ugo Pattacini" <notifications@github.com> ha scritto:

> In tests/libYARP_math/MathTest.cpp
> <https://github.com/robotology/yarp/pull/697#discussion_r54333713>:
>
> >      }
> > +
> > +    //tests written by Giulia V
>
> This comment should be removed [image: :smirk:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/robotology/yarp/pull/697/files#r54333713>.
>